tgui/packages/tgui/interfaces/RequestsConsole/Types.tsx

```diff
@@ -0,0 +1,55 @@
+import { BooleanLike } from 'common/react';
```

Comment:
a bit of a nit but this is not a component. the correct file name would be `types.ts` (lowercase).
tsx implies jsx is returned, which also warrants PascalCase in vanilla react